packages/logging_cloudwatch/amplify_logging_cloudwatch/lib/src/queued_item_store/index_db/indexed_db_adapter.dart

```diff
@@ -129,25 +129,7 @@ class IndexedDbAdapter implements QueuedItemStore {
 
   @override
   Future<Iterable<QueuedItem>> getAll() async {
-    final readValues = <QueuedItem>[];
-
-    await _databaseOpenEvent;
-    final store = _getObjectStore();
-    final request = store.getAll(null, null);
-
-    await request.future;
-
-    for (final elem in request.result) {
-      final value = elem as Map<String, dynamic>;
-      final id = value['id'] as int;
-      final itemValue = value['value'] as String;
-      final timestamp = value['timestamp'] as String;
-      readValues.add(
-        QueuedItem(id: id, value: itemValue, timestamp: timestamp),
-      );
-    }
-
-    return readValues;
+    return getCount();
```

Comment:
Why do we have 2 separate methods if they do the same exact thing?

Reply:
> Why do we have 2 separate methods if they do the same exact thing?

this is the case only for IDBObjectStore. other implementations of QueuedItemStore have different impl for getCount and getAll.

Reply:
Ah ok thanks for clarifying